src/domains/ecommerce-app/api/npg-notification/_npg_notifications_policy.xml.tpl

```diff
@@ -159,7 +159,10 @@
         <!-- end send transactions service PATCH request -->
     </inbound>
     <backend>
-        <forward-request timeout="10" />
+        <retry condition="@(context.Response.StatusCode >= 500)"
+              interval="1" count="2" first-fast-retry="true">
```

Comment:
```suggestion
              interval="2" count="3" first-fast-retry="true">
```

What about use the same configuration used for [transactions-service retry policy](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/9a895f367227770c53f403da0c6f09490a1b33a5/src/main/resources/application.yml#L4-L5) here too?

Reply:
We could do it, do we know what it the client's timeout?

Reply:
Applied in [f4c1f92]